src/xbot/common/subsystems/BasePoseSubsystem.java

```diff
@@ -47,8 +47,7 @@
     
     private BooleanProperty rioRotated;
     
-    @Inject
-    public PoseSubsystem(WPIFactory factory, XPropertyManager propManager) {
+    public BasePoseSubsystem(WPIFactory factory, XPropertyManager propManager) {
```

Comment:
If this class doesn't actually publish any telemetry data (it doesn't right now) it shouldn't be a `TelemetrySource`. A subclass can decide to implement that interface, but there's no reason to require it here.